scene/gui/menu_button.cpp

```diff
@@ -85,6 +85,16 @@ void MenuButton::show_popup() {
 		Transform2D xform = get_viewport()->get_popup_base_transform_native();
 		rect = xform.xform(rect);
 	}
+	Rect2i scr_usable = DisplayServer::get_singleton()->screen_get_usable_rect(get_window()->get_current_screen());
+	Size2i max_size = Size2i();
+	if (scr_usable != Rect2i()) {
```

Comment:
```suggestion
	if (scr_usable.has_area()) {
```
(though it's a bit different check, so might be not applicable)